src/TurbulenceConvectionUtils.jl

```
@@ -307,18 +307,17 @@ function run_SCM_handler(
     namelist["meta"]["uuid"] = uuid
     # set output dir to `tmpdir`
     namelist["output"]["output_root"] = tmpdir
+    namelist["stats_io"]["calibrate_io"] = true
 
     # run TurbulenceConvection.jl with modified parameters
-    try
-        Logging.with_logger(Logging.NullLogger()) do
-            main1d(namelist; time_run = false)
-        end
-    catch e
+    _, ret_code = Logging.with_logger(Logging.NullLogger()) do
+        main1d(namelist; time_run = false)
+    end
+    if ret_code ≠ :success
         model_error = true
         message = ["TurbulenceConvection.jl simulation $(basename(m.y_dir)) failed with parameters: \n"]
         append!(message, ["$param_name = $param_value \n" for (param_name, param_value) in zip(u_names, u)])
         @warn join(message)
-        @warn join(["This was caused by ", e])
```

Is there a way we can report the error message from TC.jl? Like this, we only know that the simulation failed and for what parameters, but not the actual error.

---

I guess this goes along the lines of what Jake was suggesting with re-throwing the error message? We can use the return code to print something, but I guess the new try-catch in TC may swallow errors.. Maybe we need to update that..

---

I think that would be really helpful.

---

This is technically an upstream issue-- we just need for TC to add this to the warn log. I'll open a PR, but I don't think that should stop this PR.